### PR TITLE
[PS-676] Accessibility - "Options" expand/collapse control does not announce state

### PR DESCRIPTION
**Pull request by andrebispo5**
## Type of change
- [X] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective
Make options button and chevron button work as a single button. Add accessibility text to describe the state of the layout.

## Code changes
* **src/App/Pages/Send/SendAddEditPage.xaml**: Moved the click event to the parent `StackLayout`. Added accessibility text to `StackLayout`. Removed accessibility on views inside `StackLayout`.
*  **src/App/Pages/Send/SendAddEditPageViewModel.cs**: Added property for the accessibility text. Raise prop change on `ShowOptions` set.

## Before you submit
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: src/App/Pages/Send/SendAddEditPage.xaml [hunk 1]
```diff
@@ -252,28 +252,33 @@
...
+                                <TapGestureRecognizer Tapped="ToggleOptions_Clicked"/>
+                            </StackLayout.GestureRecognizers>
                             <Button
                                 Text="{u:I18n Options}"
                                 x:Name="_btnOptions"
                                 StyleClass="box-row-button"
                                 TextColor="{DynamicResource PrimaryColor}"
                                 Margin="0"
-                                Clicked="ToggleOptions_Clicked"/>
+                                AutomationProperties.IsInAccessibleTree="False"/>
```
Comment (fedemkr) [inline src/App/Pages/Send/SendAddEditPage.xaml:267]: Isn't the default value of `AutomationProperties.IsInAccessibleTree ` `False`? Is there a bug or something that this line is needed?

File: src/App/Pages/Send/SendAddEditPage.xaml [hunk 2]
```diff
@@ -252,28 +252,33 @@
                         </StackLayout>
                         <StackLayout
                             Orientation="Horizontal"
-                            Spacing="0">
+                            Spacing="0"
+                            AutomationProperties.IsInAccessibleTree="True"
+                            AutomationProperties.Name="{Binding OptionsAccessilibityText}">
+                            <StackLayout.GestureRecognizers>
+                                <TapGestureRecognizer Tapped="ToggleOptions_Clicked"/>
```
Comment (fedemkr) [inline src/App/Pages/Send/SendAddEditPage.xaml:259]: Given that you're modifying this, could you change this to use `Command` instead of the event now?

**Review by fedemkr — CHANGES_REQUESTED** [review]

File: src/App/Pages/Send/SendAddEditPage.xaml (hunk 2 above)
Comment (andrebispo5) [reply src/App/Pages/Send/SendAddEditPage.xaml:259]: Sure i'll fix that.

File: src/App/Pages/Send/SendAddEditPage.xaml (hunk 1 above)
Comment (andrebispo5) [reply src/App/Pages/Send/SendAddEditPage.xaml:267]: Default value is Null. 
image

Comment (fedemkr) [reply src/App/Pages/Send/SendAddEditPage.xaml:267]: Oh ok great then 😄

**Review by fedemkr — APPROVED** [review]

**Pull request merged**